### ISSUE TITLE
Cards hidden until 'voting mode' is enabled

### ISSUE THREAD

**Issue by jenningsb2** [d0x2f/retro.tools#175]
If you're not the author of the card, you shouldn't be able to see the cards other users write until 'voting mode' has commenced. 

This helps limit groupthink. If you see others writing about a specific topic, it could steer your thought process toward their issue prematurely and limit the topic diversity of the session.

It would be great to have a setting (similar to the 'voting allowed' selection) to hide the text of cards you haven't authored.

I don't think you need to take this wildly far. Simple is better.

**Comment by d0x2f** [comment]
Good idea, I'll see if I can work on it this week.

**Comment by d0x2f** [comment]
I'm undecided on how to approach this, here are a few options in order of easiest to hardest:

- Always obscure cards when voting is disallowed.
  - This'll mean that the current style of adding cards visible to everyone won't be possible.
- Seperate `voting allowed` and `obscure cards` toggles that the board owner can flip.
  - A downside to this is that voting could be allowed at the same time that cards are obscured, which is a bit weird.
- A 3-way switch for `voting allowed`, `voting disallowed` and `voting disallowed + cards obscured`
  - This way we avoid the strange state and allow retro runners to use either style.

After all the cards are revealed, there're likely to be duplicates where people added the same thing. But there's currently no ability to merge cards, which would be an annoyance.

**Comment by d0x2f** [comment]
I think option 2 would be fine for now as well.
I'll go ahead and do that :+1:

The duplicate problem can be addressed seperately and tracked here: #3

**Comment by troyane** [comment]
Hey, @d0x2f how is the progress on this issue? 

I also found this feature request very helpful for more mindful retros.

**Comment by d0x2f** [comment]
Yes this is something I want to do, sorry it got left by the wayside as I've been focussed on other projects.
I'll see if I can pick this up again.
